Rakefile

```diff
@@ -7,7 +7,26 @@ rescue LoadError
   abort 'Please run this task using `bundle exec rake`'
 end
 
+task default: %i[test rubocop]
+
 RuboCop::RakeTask.new
-task default: %i[spec rubocop]
+
+desc 'Run the unit tests using rspec'
+task :test do
+  sh('rspec')
```

Comment:
Question: why `rspec` instead of `bundle exec rspec`? Also applies to the `yard` calls below.

Reply:
Because the Rakefile is already called inside a `bundle exec` context so in those cases every subsequent shell invocation is already in a bundler context (ie implicitly wrapped in `bundle exec` automatically).

I've discovered that fact a while ago; if you want to convince yourself, you should be able to create a dummy `task :demo { sh(' gem', 'env') }` then run `bundle exec rake demo` and see that the underlying shell command still prints the bundled ruby context 🙂